src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -61,7 +62,8 @@
  */
 public class RemoteOperationResult implements Serializable {
 
-	/** Generated - should be refreshed every time the class changes!! */;
+    /** Generated - should be refreshed every time the class changes!! */
+    ;
     private static final long serialVersionUID = 1129130415603799707L;
```

Comment:
Please, update ```serialVersionUID```. Generate a new value, see **second** answer in http://stackoverflow.com/questions/24573643/how-to-generate-serial-version-uid-in-intellij for help.